DISCLOSURE_POLICY.md

```diff
@@ -1,17 +1,17 @@
 # Responsible Disclosure Policy
 
-Bitcoin ABC takes security very seriously.  We greatly appreciate any and all disclosures of bugs and vulnerabilities that are done in a responsible manner.  We will engage responsible disclosures according to this policy and put forth our best effort to fix disclosed vulnerabilities as well as reaching out to numerous node operators to deploy fixes in a timely manner.
+Lotus takes security very seriously.  We greatly appreciate any and all disclosures of bugs and vulnerabilities that are done in a responsible manner.  We will engage responsible disclosures according to this policy and put forth our best effort to fix disclosed vulnerabilities as well as reaching out to numerous node operators to deploy fixes in a timely manner.
 
 This disclosure policy is also intended to conform to [this proposed standard](https://github.com/RD-Crypto-Spec/Responsible-Disclosure/blob/184391fcbc1bbf3c158c527a841e611ac9ae8388/README.md) with some modifications (see below).
 
 ## Responsible Disclosure Guidelines
 
-Do not disclose any bug or vulnerability on public forums, message boards, mailing lists, etc. prior to responsibly disclosing to Bitcoin ABC and giving sufficient time for the issue to be fixed and deployed.
+Do not disclose any bug or vulnerability on public forums, message boards, mailing lists, etc. prior to responsibly disclosing to Lotus and giving sufficient time for the issue to be fixed and deployed.
 Do not execute on or exploit any vulnerability.  This includes testnet, as both mainnet and testnet exploits are effectively public disclosure.  Regtest mode may be used to test bugs locally.
 
 ## Reporting a Bug or Vulnerability
 
-When reporting a bug or vulnerability, please provide the following to security@bitcoinabc.org:
+When reporting a bug or vulnerability, please provide the following to security@lotus.org: 
```

Comment:
This doesn't exist, but we can setup an email on `givelotus.org`

Reply:
Yep, didn't know what to put here but better change to non existing email so people will not write letters about Lotus to BitcoinABC😁